span_sinks.go

```diff
@@ -0,0 +1,350 @@
+package veneur
```

Comment:
nit: my gut tells me to name this `span_sink.go` (singular), not plural, since it's the file that defines (and then uses) the `SpanSink` interface. I don't have a great explanation for why the interface definition should take precedence over the definition of multiple types that use the interface when determining the name, other than "it tickles my stomach a bit to see it the other way".